stubservice/main.go

```diff
@@ -148,6 +148,8 @@ func pingdomHandler(w http.ResponseWriter, req *http.Request) {
 	attrQuery.Set("medium", "pingdom")
 	attrQuery.Set("campaign", "pingdom")
 	attrQuery.Set("content", "pingdom")
+	attrQuery.Set("funnel_experiment", "pingdom")
+	attrQuery.Set("funnel_variation", "pingdom")
```

Comment:
I'm not sure how this `pingdom` stuff works (or if it's used?), but I would guess the query params should be the `funnel_`-less versions.